tests/basic/global_vars_test/global_vars_test.c

```diff
@@ -84,16 +87,30 @@ void perform_errno_test(void)
     int l_err_no = 0;
 
     // pass a non existent file to cause failure and check errno
-    _do_file_ops("file_fake.txt");
-    l_err_no = errno;
+    l_err_no = _do_file_ops("file_fake.txt");
+
     if (l_err_no != 2)
+    {
         prog_exec_status |= (1 << en_test_errno_fail_index);
+        fprintf(
+            stdout,
+            "fake test failed: errno=%i l_err_no=%i\n",
+            errno,
```

Comment:
Do you want to keep printing errno now that you changed the test?

Reply:
I think that there is no harm in doing it. I'm still not entirely sure if this will fix the problem or not.